cerbero/config.py

```diff
@@ -449,6 +450,7 @@ def load_defaults(self):
         self.set_property('extra_bootstrap_packages', {})
         self.set_property('binaries_local', None)
         self.set_property('binaries_remote', None)
+        self.set_property('consider_parents_for_recipe_checksum', False)
```

Comment:
I would name the property by what it does and not how it does it. My  suggestion is strict_cache_invalidation (there can be potentially more things to invalidate the cache in a strict way)

Reply:
Actually, discussing the name with other team members, they proposed to use something more explicit like this rather than the `strict` name which wasn't clear for some people.